.env

```diff
@@ -1,13 +1,12 @@
-  update_interval_time = "30000"
-
+  type=service_account
+  project_id=express-sheets-trial
+  private_key_id=0e2a76e2d8c2537da1260d4f76d4b0f32217d62a
+  private_key=LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1JSUV1Z0lCQURBTkJna3Foa2lHOXcwQkFRRUZBQVNDQktRd2dnU2dBZ0VBQW9JQkFRQ2VjMkQrUml6ckxad2sKUXpXeU9nRStBbVlEekczc043RzNLN1piMG5xRlh2L2JpZCtzZ2NuKytCVTNBSGVLVzl6WU5CMUZZaFlKUHpXQwpUaCtaTm9VZndxcE0ySWtWR1VoMDNQNXk4LzFUampSWGwydzBOMzd1OFM0cVVEaUFSYnpnTUNQcW5vd2doZ3hzCnArTERYUmVyVGg0OS9LRVdUQ251NXJCd2EzOHRHNUxRbzZucnpVcXZuWEl0RWJvUEJVeGx1bU02eUVUWXlQVCsKeEpkaHQzYXIxcThVWmRzamJGakZoYnp0MzJmTTR1cFRabUNWSW9wZkxEQTR5T0VrWGw1S0ExNHhtdko4RzhndgpxaGtvdjVmSGgrMEJzS0hicDc4MUdtL1VYQnVQSDdBeTVDZER1Z3d4Z3luSHIxSGEyQ0ltS0lLYlBka0RYc2lDCmI3cnFmN1AzQWdNQkFBRUNnZ0VBQVQ4aExTby8zSFRLT3Q3T1lQTm81VjNoQ0Y5bStabW00d2RlbEordzhsOGcKUXFCWG5QczBXUGV4bUs2WUdnTytFRzU4N0hkeXFCSjZONUc0U2RnUHVxYXJzb2FiVW9KM2VManI2TkZEaXBzdApzSENJT2Q4UG9NQmVhNE9xY3liMkswRng4RTVLOGVXZW1WMS9yS0tYTE95c0xXb1U1UWVMUmdMcVRmQ1hIaVNCCldldXFadXU5bkw1RjZ4SGVOOERoOG1MeU9wam45UkNlTnhnd0s3OTlTeEtWVXR0NWloOEtVSXU1clF5QU1NOUcKSzA1WFJCZEo2KzVlVityVXZXRWFQMkpvcGUzTEFhbk5SK3FiUVY4ZlV0SWE1TTBwM1FzREpURnV3c2NhTFpJZApmZnF6SGwrMkFtcjJHakdOMnZHSngvamp4cXBjYm9TeXh0MVNnOC9VQVFLQmdRRFo5RTNFckRCMzVKNTQxWHhQClpyQkV2bDY0cVpSVUIwdUZQSU9qZkMrKyswWDFHK3pZYi9nbnZQWnpRSkRNcGdUSHVPVUJrcHdpRmJGN0dtNFAKaDZDd2xWcjVrZGl1SnlVaDl0QVBtTTF4NlloRWoyT09UdkZxdHk0bHFxUmlNU1ZlekFWbUJLcWt3SHd2WU1YeQpvNHpsMU9KSDFtemltcG9vQkdTSkdiRGF3UUtCZ1FDNkhBdDVhOEp5MWNNQnlhSElvclEzQk0xWGhKWkl2aCtFCllualpCUlhocEhvWmVxOXl1TElXSXNSUUpYTlYyUVJxVEdpUzF5azc3a3BobjNJMU1wRmtrTnM3OVIwQXhDRC8KRnZObzhMM2E2ZnYyeTByYmpBRTh5cDk3UGNnd1kxS3NlajdjckZPT1Fxd2tlQ055dGs5YXBJZFkrWFNKSzdFMgowL29UbEV4VXR3Si9aaW9lU2hKc3BjYUVnaCtRSmFmOCtEeEVLa3RUUEJBdlZrVUhiNXJHeHI4Q0MrTUxJbWxaCjZVYXFQMXdFZytsT1ZGQlhDR1d3eWtsZzZGTjdROHhQaG5aOExvQXZFWE5IOUtPODQ1Szh1cWk5TjJYL1JMVzgKZEVscFdENmt6Mkk2SWpXQVVsSGRxMXlPMmFhREl6bUcwalQzM0xTbVZaSWlzSjllZ09XS0FRS0JnRHFlVHJibAp6QThqYVJLTkVab0NOVVVIREM0K29wOHkwdlo5bFphMVZWa0cxelpqUUJPZ1BPZGY5L2xPY205ZHRWaE56THBECmJzZGpuS1gyOUovdzEzNlhhWllPOUIyanFNanZ6NmNEL0tpUS9qMlRkbmFFT0hlUFVvQWN5NHNDSGE0WnF1Y1MKbUJhOHhUK1BYNHJvQlVLRmcrUHdYbFkrSHNrb0p5STNtWHN0QW9HQUFUNXc3cWdCZklFRGl2ZXovaUJoa2FCSwp0eCtabXRSakF1OFdlaTBUcVJjc1dIbytDWDZBb1hMZnIvZ3V4cEJKTzhqK3BqTkJRK0FaM2szUHlScTRjM0ZZCngzNGZwMk9Rbi9MNCtXL1NERTJMZS96cTRleFI4QzN5MUIwV25tb2FtMVIxZEJMUDVqWnV4Q3JnTHIrd3AyM00KTng0QlVQRWxLMmJPWVF2Z2JJaz0KLS0tLS1FTkQgUFJJVkFURSBLRVktLS0tLQo=
+  client_email=google-spreadsheet-access@express-sheets-trial.iam.gserviceaccount.com
+  client_id=118109966120154911750
+  auth_uri=https://accounts.google.com/o/oauth2/auth
+  token_uri=https://oauth2.googleapis.com/token
+  auth_provider_x509_cert_url=https://www.googleapis.com/oauth2/v1/certs
+  client_x509_cert_url=https://www.googleapis.com/robot/v1/metadata/x509/google-spreadsheet-access%40express-sheets-trial.iam.gserviceaccount.com
+  sheet_id=1GvlGcn-IHnacCAztYzIENNIZs4bc_Ys4zVcEM94LPFU
+  update_interval_time=30000
```

Comment:
We should not commit this in git as this is going to be public repository.

Reply:
this is a sample spreadsheet created for this purpose... but maybe the secret keys should be passed on from env, I agree